### PR TITLE
Add 10216 Explore the Swift on Server ecosystem

### PR DESCRIPTION
**Pull request by dl-alexandre**

### CODE REVIEW

File: Sources/WWDCNotes/WWDCNotes.docc/WWDC24/WWDC24-10217-Explore-Swift-performance.md [hunk 1]
```diff
@@ -12,7 +12,11 @@ Discover how Swift balances abstraction and performance. Learn what elements of
    }
 }
 
-😱 "No Overview Available!"
+## Key takeaways
```
Comment (Jeehut) [inline Sources/WWDCNotes/WWDCNotes.docc/WWDC24/WWDC24-10217-Explore-Swift-performance.md:15]: This note seems to be empty? Also images were added to 10216 but this note is 10217?

**Review by Jeehut — CHANGES_REQUESTED** [review]
Left a comment in one place, probably just an oversight. Everything else looks good, thank you!

**Review by dl-alexandre — COMMENTED** [review]
Forgive me I'm not finding the comment in 10200. 10216 is completed as the server-side note. I've only started on 10217. My apologies for the confusion.

**Review by Jeehut — APPROVED** [review]
Thank you for providing 3 notes at once! 🥳

**Pull request merged**